ts/output/common/Wrappers/mtable.ts

```diff
@@ -681,7 +681,7 @@ export function CommonMtableMixin<
         //  Stretch the stretchable children
         //
         for (const child of stretchy) {
-          child.coreMO().getStretchedVariant([W]);
+          child.coreMO().getStretchedVariant([Math.max(W, child.getBBox().w) / child.coreRScale()]);
```

Comment:
ditto

Reply:
Same here.  The `child.coreRScale()` can be different for each child.  This finds the relative scaling of the core `mo` in each child, and since some children might be embellished operators with several changes of relative scaling on the way, and others might not, the scaling value has to be computed for each child separately.